project_euler/problem_119/sol1.py

```diff
@@ -0,0 +1,44 @@
+"""
+"https://projecteuler.net/problem=119"
+
+Name: Digit power sum
+
+The number 512 is interesting because it is equal to the sum of its digits
+raised to some power: 5 + 1 + 2 = 8, and 83 = 512. Another example of a number
+with this property is 614656 = 284. We shall define an to be the nth term of
+this sequence and insist that a number must contain at least two digits to have a sum.
+You are given that a2 = 512 and a10 = 614656. Find a30
+"""
+
+
```

Comment:
```suggestion
```

Reply:
removed the empty line.

Reply:
Remove both the newlines before the imports :)
You could've directly applied the suggestion but it's up to you.